<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能仓储管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ip_label">
          <property name="text">
           <string>服务器地址：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="ip_lineedit">
          <property name="text">
           <string>192.168.1.164</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dataport_label">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>数据端口：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="dataport_lineedit">
          <property name="text">
           <string>6969</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="videoport_label">
          <property name="text">
           <string>摄像头端口：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="videoport_lineedit">
          <property name="text">
           <string>6868</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="connect_pushbutton">
          <property name="text">
           <string>连接服务器</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="openvid_pushbutton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>打开仓库监控</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="openwrhouse_pushbutton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>打开仓库</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="opendata_pushbutton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>开启数据获取</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closedata_pushbutton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>关闭数据获取</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="tempmax_label">
          <property name="text">
           <string>温度上限：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="hummax_label">
          <property name="text">
           <string>湿度上限：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="tempmax_spinbox">
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="hummax_spinbox">
          <property name="value">
           <number>32</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="tempmin_label">
          <property name="text">
           <string>温度下限：</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="hummin_label">
          <property name="text">
           <string>湿度下限：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="tempmin_spinbox">
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="hummin_spinbox">
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="message_label">
          <property name="text">
           <string>短信内容：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="message_lineedit">
          <property name="text">
           <string>temp high!!!!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="data_groupbox">
        <property name="title">
         <string>节点数据</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLabel" name="light_label">
             <property name="text">
              <string>光照：0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="hum_label">
             <property name="text">
              <string>湿度：0    </string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="temp_label">
             <property name="text">
              <string>温度：0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="adc_label">
             <property name="text">
              <string>ADC ：0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="x_label">
             <property name="text">
              <string>X：0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="y_label">
             <property name="text">
              <string>Y：0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="z_label">
             <property name="text">
              <string>Z：0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ctrl_groupbox">
        <property name="title">
         <string>设备控制</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="onled_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>打开LED</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="offled_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>关闭LED</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="onfan_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>打开风扇</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="offfan_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>关闭风扇</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="onsevenled_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>打开数码管</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="offsevenled_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>关闭数码管</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="onspeaker_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>打开蜂鸣器</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="offspeaker_pushbutton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>关闭蜂鸣器</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="state_groupbox">
        <property name="title">
         <string>设备状态</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="led1state_label">
           <property name="text">
            <string>LED状态   ：未知</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fanstate_label">
           <property name="text">
            <string>风扇状态  ：未知</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sevenledstate_label">
           <property name="text">
            <string>数码管状态：未知</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="speakerstate_label">
           <property name="text">
            <string>报警器状态：未知</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>196</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
